test/lib/grist-api.ts

```diff
@@ -0,0 +1,283 @@
+/**
+ * The test is intended to test the behavior of the library, i.e. translating python calls to HTTP
```

Comment:
`translating javascript calls to HTTP requests`

Reply:
Oops, thanks!